docs/restic.md

```diff
@@ -1,23 +1,31 @@
 # Restic Integration
 
-Ark now has support for backups/restores of pod volume data using [restic][1], an open-source tool for doing
-backups and restores of filesystem data. This enables you to take backups of additional Kubernetes volume 
-types beyond those with snapshot APIs that are integrated with Ark (i.e. AWS EBS, GCP PD, Azure PD). It also lays 
-the foundation for future work to support cross-cloud stateful migrations. Note that the details of this feature,
-including names, commands, etc., may change as we receive feedback and refine our implementation. 
-
-Two new Ark custom resources have been created to support this feature: `PodVolumeBackup` and `PodVolumeRestore`.
-Additionally, a new Ark daemonset has been created that runs two controllers, one for each of the two new CRDs, on
-each node in the cluster. When an Ark backup is created that includes pods annotated for restic backup, the main Ark 
-backup controller will create a `PodVolumeBackup` custom resource that's owned by the `Backup`. The pod volume backup
-controller running on the pod's node will observe the new custom resource, and will run a restic backup of the volume 
-(accessing the volume's data via a hostPath mount of `/var/lib/kubelet/pods`). The main Ark backup controller will 
-wait for the `PodVolumeBackup` to complete before completing the Ark backup. Restores proceed similarly with some 
-minor differences to account for the fact that a new pod/volume is being created.
+As of v0.9.0, Ark has support for backing up and restoring Kubernetes volumes using a free open-source backup tool called 
+[restic][1].
+
+Ark has always allowed you to take snapshots of persistent volumes as part of your backups, provided you’re using one of 
+the supported cloud providers’ block storage offerings (Amazon EBS Volumes, Azure Managed Disks, Google Persistent Disks). 
+And, since v0.6.0, we’ve had a plugin model which enables anyone to easily implement additional object and block storage 
+backends, outside of the main Ark repository.
+
+We integrated restic with Ark so that users have an out-of-the-box solution for backing up and restoring any type of Kubernetes
+volume*. We view this as a new capability for Ark, rather than a replacement for existing functionality. If you're running on AWS, and
+taking EBS snapshots as part of your regular Ark backups, great! There's no need to switch to using restic. However, if you've
+been waiting for a snapshot plugin to be implemented for your storage platform, or you're using EFS, AzureFile, NFS, emptyDir, 
+local, or any other volume type that doesn't have a native snapshot concept, restic may be for you.
+
+Additionally, since restic is not tied to a specific storage platform, this integration paves the way for future work to enable
+cross-volume-type data migrations. Stay tuned as this evolves!
+
+*<small> hostPath volumes are not supported, but the [new local volume type][4] is!</small>
```

Comment:
Not sure if you're intending on linking to the Github version of this file, but using their view button on the diff doesn't render the text any smaller.